src/shared/components/base/gokb-textarea-field/gokb-textarea-field.vue

```diff
@@ -86,7 +87,7 @@
     },
     data () {
       return {
-        defaultRows: 1
```

Comment:
`defaultRows` is still referenced. We could just write `:rows="1"` or capsule it in the `data` like it was before.